language-server/modules/langserver-core/src/test/java/org/ballerinalang/langserver/codeaction/CodeActionPerformanceTest.java

```diff
@@ -47,6 +52,12 @@ public void test(String config) throws IOException, WorkspaceDocumentException {
 
     @Override
     public String getResponse(Path sourcePath, Range range, CodeActionContext codeActionContext) {
+        // Waiting for the server to load the packages from the distribution and central
+        try {
+            Thread.sleep(20000);
```

Comment:
We can get this tests to pass without using a sleep by mocking the `loadBallerinaxPackagesFromCentral` method. See https://github.com/ballerina-platform/ballerina-lang/blob/84ad82802f7a233f35df2d125ed5e766af292c2c/language-server/modules/langserver-core/src/test/java/org/ballerinalang/langserver/AbstractLSTest.java#L100. 

We should add a mock for this method as well.